charts/redpanda/values.yaml

```diff
@@ -271,6 +271,8 @@ monitoring:
   enabled: false
   scrapeInterval: 30s
   labels: {}
+  enableHttp2: true
```

Comment:
why enable this true by default? Should we opt in or is the default value true in the resource spec?

Reply:
I can remove it, there is no default value https://github.com/prometheus-operator/prometheus-operator/blob/main/Documentation/api.md#endpoint will using `with` suffice to see if its present? 

```
{{- with .Values.monitoring.enableHttp2 }}
enableHttp2: {{ . }}
{{- end }}
```

Reply:
The question is about the values file and the default value you set for it. Your suggestion makes sense, we can also try

```
{{- if dig "enableHttp2" "" .Values.monitoring }}
enableHttp2: .Values.monitoring.enableHttp2
{{- end }}
```

 and let's then comment it out on the values file, but with a note above about how to use it. Thank you,